docs/providers/kubernetes/overview.md

```diff
@@ -28,22 +28,22 @@ section of the answers.conf file. An example is below:
 namespace: mynamespace
 ```
 
-#### providerconfig
+#### provider-config
 
-For Kubernetes the configuration file as specified by `providerconfig`
+For Kubernetes the configuration file as specified by `provider-config`
 is optional. Hosts that have kubernetes set up and running on them
-may not need a `providerconfig` to be specified because kubernetes
+may not need a `provider-config` to be specified because kubernetes
 services are listening on default ports/addresses. However, if
 kubernetes was set up to listen on different ports, or you wish to
 connect to a remote kubernetes environment, then you will need to
 specify a location for a provider config file.
 
-One example of specifying a `providerconfig` is below:
+One example of specifying a `provider-config` is below:
 
 ```
 [general]
 provider: kubernetes
-providerconfig: /host/home/foo/.kube/config
+provider-config: /host/home/foo/.kube/config
```

Comment:
can we also get this updated to remove `/host/` from this ? separate commit would be fine.. I think there was a time where we didn't do the conversion so the user had to add it themselves